VirtoCommerce.Platform.Web/Licensing/LicenseProvider.cs

```diff
@@ -0,0 +1,24 @@
+using System.Web.Hosting;
+
+namespace VirtoCommerce.Platform.Web.Licensing
+{
+    public static class LicenseProvider
+    {
+        public static License LoadLicense()
+        {
+            License license = null;
+            var licenseFilePath = HostingEnvironment.MapPath(Startup.VirtualRoot + "/App_Data/VirtoCommerce.lic");
+            if (System.IO.File.Exists(licenseFilePath))
+            {
+                var rawLicense = System.IO.File.ReadAllText(licenseFilePath);
+                license = License.Parse(rawLicense);
+
+                if (license != null)
+                {
+                    license.RawLicense = null;
+                }
+            }
+            return license;
+        }
+    }
```

Comment:
Not testable code. Violation of the SOLID principles. What if we want to get the license from a different source?
I'm really hoping that we'll fix violations like this in future )